explainer-server/app/views/explainEditor.scala.html

```diff
@@ -48,14 +51,23 @@
         function updateWordCountDisplay(){
             $(".top-toolbar__word-count-wrapper__count").text(getBodyWordCount())
         }
-        function updateWordCountWarningDisplay(){
-            if(getBodyWordCount()>maxWordCount){
+        function updateWordCountWarningDisplay() {
+            if (getBodyWordCount() > maxWordCount) {
                 $(".top-toolbar__word-count-wrapper__warning-outer").show();
-            }else{
+            } else {
                 $(".top-toolbar__word-count-wrapper__warning-outer").hide();
             }
         }
 
+        function updateStatusBar(message){
+            if(message.length>0){

```

Comment:
This could be simplified to `if(message.length) {`... JS evaluates `0` to `false`.